src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -54,6 +55,23 @@ const FacebookButton = styled(Button)`
   }
 `;
 
+const GoogleButton = styled(Button)`
+  &&& {
+    background-color: white;
+    border: 1px solid #4285f4;
+    width: 320px;
+    color: #4285f4;
+    span {
+      position: absolute;
+      inset: 0px auto 0px 22px;
+      width: 18px;
+    }
+    &:hover {
+      box-shadow: 0 0 6px #4285f4;
+    }
```

Comment:
I see that the other button isnt having any shadow, perhaps for consistency we remove this shadow as well?